DependencyInjection/Configuration.php

```diff
@@ -69,7 +69,7 @@ public function setDeprecatedNode($node, $type, $name, $deprecatedMessage)
         $node = $node->$type($name);
 
         if (Kernel::VERSION_ID >= 30400) {
-            $node = $node->setDeprecated($deprecatedMessage);
+            $node = $node->setDeprecated('mmucklo/queue-bundle', '5.1',$deprecatedMessage);
```

Comment:
So symfony < 5.1 will use the old syntax, so this will need a separate

```php
if (Kernel::VERSION_ID >= 51000) { 
```

wrapped around it if I'm not mistaken and put the original code under an else {

[Symfony 5.0 setDeprecated](https://github.com/symfony/symfony/blob/80e38d27ee145f6d0f4f2fa554a4f85ca634f729/src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php#L167)

Reply:
Ok, I see. I'll change it.

Reply:
Please review.